src/app/routes/cpsAsset/getInitialData/index.js

```diff
@@ -32,16 +35,24 @@ const transformJson = async json => {
   try {
     const formattedPageData = formatPageData(json);
     const optimoBlocks = await convertToOptimoBlocks(formattedPageData);
-    return processOptimoBlocks(optimoBlocks);
+
+    // Check for media availability and show a placeholder if media is absent.
+    const pageType = path(['metadata', 'type'], optimoBlocks);
+    const processedMediaBlocks =
+      pageType === MEDIA_ASSET_PAGE
+        ? processUnavailableMedia(optimoBlocks)
```

Comment:
Might be nice to get this into the transformer list, as that's where people will be looking for the transformers

Could be something like this:
```
const processOptimoBlocks = pipe(
  only('MAP', processUnavailableMedia),
  addHeadlineBlock,
  addSummaryBlock,
  augmentWithTimestamp,
  addBylineBlock,
  addIdsToBlocks,
  applyBlockPositioning,
  cpsOnlyOnwardJourneys,
);
```

With `only` being something like:
```
const only = (pageType, transformer) => (pageData, ...args) => {
  const isCorrectPageType = path(['metadata', 'type'], pageData) === pageType;
  return isCorrectPageType ? transformer(pageData, ...args) : pageData;
}
```